src/components/PrimaryButton.jsx

```diff
@@ -6,9 +6,13 @@ import styled, { css } from "styled-components";
  * @prop {function} onClick - Handles the action to be taken when the button is clicked
  * @param {{text: string, onClick: function}} props
  */
-export default function PrimaryButton({ text, onClick, variant, disabled, positioned, exit, underTable, toggle, blue, className }) {
+export default function PrimaryButton({ text, onClick, variant, disabled, positioned, exit, underTable, toggle, blue, className, left_align=false }) {
 
-  return (
+  return left_align ? (<LeftButton className={className} variant={variant} disabled={disabled} positioned={positioned} exit={exit} underTable={underTable} toggle={toggle} blue={blue} onClick={() => onClick()}>
+  <ButtonText variant={variant} disabled={disabled}>
+    {text}
+  </ButtonText>
+  </LeftButton> ) : (
```

Comment:
do we need to render this button with different alignment as a conditional copy? looking at the rules enforced it seems like the same effect could be from adding a prop to primary button that changes the alignment or modifying that specific table at the container level. minor since it does look better, not sure about different sized viewports though

Reply:
Yeah, a prop would be more efficient